build/sbt

```diff
@@ -66,7 +66,7 @@ Usage: $script_name [options]
   -sbt-dir   <path>  path to global settings/plugins directory (default: ~/.sbt)
   -sbt-boot  <path>  path to shared boot directory (default: ~/.sbt/boot in 0.11 series)
   -ivy       <path>  path to local Ivy repository (default: ~/.ivy2)
-  -mem    <integer>  set memory options (default: $sbt_mem, which is $(get_mem_opts $sbt_mem))
+  -mem    <integer>  set memory options (default: $sbt_default_mem, which is $(get_mem_opts $sbt_default_mem))
```

Comment:
Some of this is copied from the main sbt script. I'd rather not customize it further unless needed. 
Looks like this is already set in sbt-launch-lib.bash ... _if_ memory settings were specified. I agree that it doesn't then show up here. Honestly, just write in "2048" here in place of `$sbt_mem`

Reply:
+1 for @srowen 's advice.

Reply:
I also copied this from the latest stable sbt script

Reply:
OK, it's not a big deal to reference the declaration in the other file. But what are the other changes from?

Reply:
they also from the new sbt script

Reply:
If this PR aims to resync with `SBT 1.3.4`, shall we remove `get_mem_opts` function and `, which is $(get_mem_opts $sbt_default_mem)`  part completely.

**SBT 0.13.18**
```
$ ./sbt -h | grep mem
  -mem    <integer>  set memory options (default: 1024, which is -Xms1024m -Xmx1024m -XX:ReservedCodeCacheSize=128m -XX:MaxMetaspaceSize=256m)
```

**SBT 1.3.4**
```
$ bin/sbt -h | grep mem
  --mem    <integer>  set memory options (default: 1024)
```

SBT 1.3.4 doesn't have `get_mem_opts` at all.

Reply:
Not really aiming to resync, `declare sbt_default_mem=1024` is exists in the script `sbt-launch-lib.bash` of `sbt 0.13.18`,  we might have not copied it before.

Reply:
```
declare init_sbt_version="0.13.18"
declare sbt_default_mem=1024
```

Reply:
This doesn't update to 1.3.4 at all, no. (Moving to sbt 1.x is a fairly significant change; I couldn't get it to work a year ago) The script here is bits and pieces of an old sbt 0.x script. This is just fixing the mem message, and happens to mimic some lines from a newer sbt script.

Incidentally, I kind of would like to remove the build/mvn and build/sbt scripts anyway. It works fine with recent mvn and I don't think build tools are hard to install for developers. Although installing sbt locally installs a new version, it still does understand the 0.13-based build. I don't know how much people are attached to this 'built in' wrapper, and I don't feel strongly about.

Of course I'd also like to stop supporting 2 builds and would remove sbt. But that may wait until we face problems keeping the current one working with Scala 2.13.x or something.

These are both quite separate issues.

Reply:
What I meant was the updated code came the upstream~ I know this is not 1.3.8.
I'm also +1 for removing these wrappers. That's the reason to be reluctant to invest(make a commit on this) here.